modopt/math/metrics.py

```diff
@@ -95,7 +95,7 @@ def ssim(test, ref, mask=None):
     """
 
     if not import_skimage:  # pragma: no cover
-        raise ImportError('Scikit-Image package not found')
+        raise ImportError('Required version of Scikit-Image package not found')
```

Comment:
Maybe specify which version is expected (probably a redundancy if specified in the reqs or else, but still user-friendly)